protocol/messenger.go

```diff
@@ -4989,7 +4989,13 @@ func (m *Messenger) MuteChat(request *requests.MuteChat) (time.Time, error) {
 		return time.Time{}, err
 	}
 
-	return m.muteChat(chat, contact, MuteTill)
+	muteTillTimeRemoveMS, err := time.Parse("2006-01-02T15:04:05Z", MuteTill.Format("2006-01-02T15:04:05Z"))
```

Comment:
The suffix convention is usually `Ms`, since millisecond is a single word. `MS` looks like Microsoft ;)

Reply:
Okay, Thanks Icaro!